@commitlint/ensure/src/index.test.ts

```diff
@@ -4,7 +4,11 @@ import {camelCase, values} from 'lodash';
 import * as ensure from '.';
 
 test('exports all checkers', async () => {
-	const expected = (await glob('*.ts')).map(f => camelCase(f)).sort();
+	const ignore = ['types'];
+	const expected = (await glob('*.ts'))
+		.map(f => camelCase(f))
+		.sort()
+		.filter(item => !ignore.includes(item));
 	const actual = Object.keys(ensure).sort();
 	expect(actual).toEqual(expected);
```

Comment:
```suggestion
	expect(actual).toMatchObject(expected);
```

and you should be able to remove sorting

```ts
test('exports all checkers', async () => {
	const ignore = ['types'];
	const expected = (await glob('*.ts'))
		.map(f => camelCase(f))
		.filter(item => !ignore.includes(item));
	const actual = Object.keys(ensure);
	expect(actual).toMatchObject(expected);
});
```

Reply:
Good to know! I go with the current approach as this is supposed to test if we forgot to reexport any checker from the directory. So failing for  a new file that is not exported from `index.js` is the expected behavior